cpu/BUILD

```diff
@@ -79,7 +79,7 @@ constraint_value(
 )
 
 constraint_value(
```

Comment:
Can you move this to just above where the existing alias is today. That will keep the sorting better.